src/redux/actions/action.js

```diff
@@ -0,0 +1,25 @@
+export const FETCH_CONTINENT = 'react-capstone-project/action/FETCH_CONTINENT';
+export const FETCH_CONTINENT_FAILURE = 'react-capstone-project/action/FETCH_CONTINENT_FAILURE';
+
+export const FETCH_COUNTRY = 'react-capstone-project/action/FETCH_COUNTRY';
+export const FETCH_COUNTRY_FAILURE = 'react-capstone-project/action/FETCH_COUNTRY_FAILURE';
+
+export const fetchContinent = (payload) => ({
+  type: FETCH_CONTINENT,
+  payload,
+});
+
+export const fetchContinentFailure = (error) => ({
+  type: FETCH_CONTINENT_FAILURE,
+  payload: error,
+});
+
+export const fetchCountryDetails = (payload) => ({
+  type: FETCH_COUNTRY,
+  payload,
+});
+
+export const fetchCountryFailure = (error) => ({
+  type: FETCH_COUNTRY_FAILURE,
+  payload: error,
+});
```

Comment:
- [ ] Make sure that the right arrow icon is displaying something after clicking on the image.

Reply:
Hi @Emmyn5600,

Kindly be more specific with these changes because this is my action file and there is No right arrow here.

Thank you for your feedback.